src/sentry/web/frontend/accounts.py

```diff
@@ -121,9 +122,18 @@ def recover_confirm(request, user_id, hash):
 
 
 @login_required
+@require_http_methods(["POST"])
```

Comment:
Let me know if this still needs updated - it's showing up as a requested change.